src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterPluginUnitTest.java

```diff
@@ -28,7 +28,7 @@
                 { LibraryReadFilter.class.getSimpleName(),
                         (Consumer<SetupTest>) this::setupLibraryTest, "--library", "Foo" },
                 { MappingQualityReadFilter.class.getSimpleName(),
-                        (Consumer<SetupTest>) this::setupMappingQualityTest, "--mappingQuality", "255" },
+                        (Consumer<SetupTest>) this::setupMappingQualityTest, "--minimumMappingQuality", "255" },
```

Comment:
We should add a test case here that uses a better example for mapping quality (not 255).